qa/tasks/vstart_runner.py

```diff
@@ -333,7 +333,26 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
 
         # Filter out helper tools that don't exist in a vstart environment
         args = [a for a in args if a not in {
-            'adjust-ulimits', 'ceph-coverage', 'timeout'}]
+            'adjust-ulimits', 'ceph-coverage'}]
```

Comment:
Ah! You found it. I think the `timeout` command is standard everywhere. You don't need the fitlering below. Just remove it here.

Reply:
Yeah, fixed it.
Thanks @batrick